### DISCUSSION TITLE
how to configure a simple graph in insight dashboard?

### DISCUSSION THREAD

**Discussion by jnaav** [directus/directus#9258] [Archive]
I have a database with a field for the date (for each day) and another for data (a value that grows by day). Every day a new record is inserted.
How do I have to configure it to show me the graph of the progression in time (time series)?

I have tried multiple combinations of group aggregation and group precision but the chart is always empty. 

I checked with postman and the result is always empty. Event a simple query with this 

&groupBy[]=day(date_created)

only returns null.

**Comment by benhaynes** [comment]
That's odd, I have several Time Series panels working... they just plot a value against a timestamp. See config below, which plots our GitHub stars over time.

![Screen Shot 2021-10-30 at 3 41 17 PM](https://user-images.githubusercontent.com/522079/139556519-37344954-84f4-4c3f-98c9-cc1a67d82e5c.png)

**Comment by jnaav** [comment]
I make some test, but nothing. This is my data, is something wrong?

Tried with DateTime and Timestamp, one row by each day:
![imagen](https://user-images.githubusercontent.com/61027944/139567415-60a3bab6-c25e-4807-b6b3-3eea91a65ff9.png)

Tried multiple configurations:
![imagen](https://user-images.githubusercontent.com/61027944/139567425-ea93ce1a-30b4-47b4-a229-7a17fafa7d81.png)

But the result is always empty:
![imagen](https://user-images.githubusercontent.com/61027944/139567430-c26be232-55eb-426f-96b6-9519a46d90ff.png)

**Reply by joselcvarela** [reply]
Hello @goliath888 
It seems this only happens on the first time.
Can you try edit the chart and check if the `Value Field` is still correct?
Also, try to reload the page and then create a new chart.

**Comment by jnaav** [comment]
Hello @joselcvarela 
Tried that multiple times. Sometimes as you say, value field is empty. 
Also tried creating and deleting multiple times the chart. Saving, reloading the page, but nothing works. Tried in firefox, chrome and edge.

**Reply by joselcvarela** [reply]
I was able to reproduce your issue once, on the panel setup. But then I was able to modify it.
Also make sure you range includes your data.
If that does not work, maybe you could open an issue providing the steps to reproduce it.

**Reply by uebmaster** [reply]
I have exactly the same case.

I am trying to create a time series that show me how many items have been created in a time period (10 months by example) in the collection called estudiantes, but always i got nothing, just an empty graph.

Here are the graph's settings:

(in value field i leave default cause i only can select a "sort" option)

(in group aggregation i always select "count", i changed just to try cause i saw the example of ben)

![image](https://user-images.githubusercontent.com/5565506/144276833-ce195ac4-ac56-454e-b855-a345673a972e.png)

![image](https://user-images.githubusercontent.com/5565506/144276953-a8b5fba3-e751-445e-9677-fa2b30c16488.png)

Thanks in advance.

**Reply by jnaav** [reply]
@uebmaster If you found any way to do it, let me know please.

**Comment by jnaav** [comment]
It seems that with the latest version issue has been solved
